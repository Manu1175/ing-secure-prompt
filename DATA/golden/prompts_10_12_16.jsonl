{"id": "prompts_10_12_16:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an escalation note: Transaction Monitoring is Deprecated but was last executed 2025-09-15, Fully Automated, owned by Finance Team, Weekly cadence. Flag risk of dormant alerts.", "expected_scrub": "Write an escalation note: <PROCESS_NAME> is <PROCESS_STATUS> but was last executed <LAST_EXECUTED_DATE>, <AUTOMATION_STATUS>, owned by <RESP_TEAM>, <FREQUENCY> cadence.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Log a risk that Credit Scoring is Under Review (owner Customer Support, Fully Automated, On Demand, last run 2025-09-03). Note potential impact on downstream Loan Origination (Weekly, Semi-Automated, last run 2025-08-18, status Active).", "expected_scrub": "Log a risk that <PROCESS_NAME> is <PROCESS_STATUS> (owner <RESP_TEAM>, <AUTOMATION_STATUS>, <FREQUENCY>, last run <LAST_EXECUTED_DATE>). Note potential impact on <PROCESS_NAME> (<FREQUENCY>, <AUTOMATION_STATUS>, last run <LAST_EXECUTED_DATE>, status <PROCESS_STATUS>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a brief exec email summarizing status: Customer Onboarding (Deprecated, last 2025-09-05), KYC Verification (Deprecated, Monthly, last 2025-09-14), AML Screening (Deprecated, On Demand, last 2025-08-31), Fraud Detection (Active, Daily, last 2025-08-23).", "expected_scrub": "Draft a brief exec email summarizing status for <PROCESS_NAME> (<PROCESS_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an SOP step that, when Customer Complaint Handling (owner Customer Support, Fully Automated, On Demand) is Active (last run 2025-09-11), triage auto-tags tickets 'PRIORITY-CUST'.", "expected_scrub": "Write an SOP step that, when <PROCESS_NAME> (owner <RESP_TEAM>, <AUTOMATION_STATUS>, <FREQUENCY>) is <PROCESS_STATUS> (last run <LAST_EXECUTED_DATE>), triage auto-tags 'PRIORITY-CUST'.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a change request to align Data Backup (Under Review, Manual, On Demand, last run 2025-09-06) with Interest Calculation (Active, Fully Automated, Daily, last run 2025-09-02): ensure restore tests before EOM closes.", "expected_scrub": "Write a change request to align <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) with <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a controls snapshot comparing Regulatory Reporting (Under Review, Semi-Automated, On Demand, last 2025-09-16) and Internal Audit (Under Review, Fully Automated, Daily, last 2025-09-15).", "expected_scrub": "Compose a controls snapshot comparing <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) and <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Build a mini decision tree for Password Reset (Active, Semi-Automated, Daily, last executed 2025-08-22): if failures >3/day, auto-assign HR L2 and open P1.", "expected_scrub": "Build a mini decision tree for <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>): if failures >3/day, auto-assign L2 and open P1.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a weekly release plan for Mobile App Update (Active, Fully Automated, Weekly, last 2025-09-15) with a dependency check on Regulatory Reporting (Under Review, On Demand).", "expected_scrub": "Draft a weekly release plan for <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) with dependency check on <PROCESS_NAME> (<PROCESS_STATUS>, <FREQUENCY>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Send an activation email for CUST-0023 (Line of Credit Agreement) to Emily Williams (emily.williams@hotmail.com) — Status Active; Amount 353,236.74.", "expected_scrub": "Send an activation email for <AGREEMENT_ID> (<AGREEMENT_TYPE>) to <PARTY_NAME> (<PARTY_EMAIL>) — Status <STATUS>; Amount <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a termination summary for CUST-0002 (Letter of Credit) — Michael Clark (michael.clark@gmail.com), Status Terminated, Amount 499,667.34.", "expected_scrub": "Create a termination summary for <AGREEMENT_ID> (<AGREEMENT_TYPE>) — <PARTY_NAME> (<PARTY_EMAIL>), Status <STATUS>, Amount <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "List active supplier agreements of outsourcing/related types: SUP-0012 (Outsourcing) 64,145.18; SUP-0036 (Outsourcing) 396,361.85; SUP-0039 (Outsourcing) 61,399.13; SUP-0034 (Services Supply) 237,777.45; SUP-0024 (Goods Supply) 106,623.59.", "expected_scrub": "List active supplier agreements of specified types with IDs and amounts: <AGREEMENT_ID> (<TYPE>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a pending-action tracker: CUST-0044 (Letter of Credit) 310,738.37; CUST-0045 (Bank Guarantee) 212,569.00; CUST-0046 (Personal Loan) 451,133.43; CUST-0054 (Safe Deposit Box) 203,538.70; CUST-0055 (Credit Card) 279,479.12; CUST-0056 (Overdraft) 492,879.62.", "expected_scrub": "Prepare a pending-action tracker: <AGREEMENT_ID> (<AGREEMENT_TYPE>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a risk note flagging high amounts: CUST-0028 (Active) 497,139.00; SUP-0016 (Expired) 496,703.00; SUP-0038 (Expired) 497,735.94; CUST-0002 (Terminated) 499,667.34.", "expected_scrub": "Draft a risk note flagging high amounts: <AGREEMENT_ID> (<STATUS>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a one-paragraph engagement snapshot for Emily Williams: Mortgage (Borrower, Active; last interaction 2025-08-11) and Current Account (Holder, Active; last 2025-07-30).", "expected_scrub": "Write a one-paragraph engagement snapshot for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>; last <LAST_INTERACTION_DATE>) and <PRODUCT_2> (...).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a micro-segmentation note for Mobile Banking Active users: include Robert Williams (last interaction 2025-08-08), Alex Garcia (2025-06-24), Katie Smith (2025-05-18). Suggest push notification campaign.", "expected_scrub": "Produce a micro-segmentation note for <PRODUCT> <STATUS> users: include <FIRST_NAME> <LAST_NAME> (last <LAST_INTERACTION_DATE>), ... Suggest a campaign.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a cross-sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Propose a mortgage health check offer.", "expected_scrub": "Draft a cross-sell note for <FIRST_NAME> <LAST_NAME>: <PRODUCT_A> (<RELATIONSHIP>, <STATUS>), <PRODUCT_B> (<RELATIONSHIP>, <STATUS>), <PRODUCT_C> (<RELATIONSHIP>, <STATUS>). Propose an offer.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose gentle nudges to re-engage Online Banking users now Inactive: John Taylor (last 2025-03-19), Katie Taylor (2025-05-04).", "expected_scrub": "Compose gentle nudges to re-engage <PRODUCT> users now <STATUS>: <FIRST_NAME> <LAST_NAME> (last <LAST_INTERACTION_DATE>), ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a relationship matrix entry for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).", "expected_scrub": "Create a relationship matrix entry for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>), <PRODUCT_2> (...), <PRODUCT_3> (...).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a compliance linkage note: Emily Jones has Electronic Banking Agreement CUST-0050 Active (Amount 35,037.61). Product links show Mobile Banking (User, Active) and Current Account (Holder, Active).", "expected_scrub": "Compose a compliance linkage note: <FIRST_NAME> <LAST_NAME> has <AGREEMENT_ID> <STATUS> (Amount <AMOUNT>). Product links show <PRODUCT_1> (<RELATIONSHIP>, <STATUS>) and <PRODUCT_2> (<RELATIONSHIP>, <STATUS>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a friendly welcome email to Jane Smith for her Mortgage (Borrower, Active; start 2024-09-30). Invite her to set up Online/Mobile Banking.", "expected_scrub": "Write a friendly welcome email to <FIRST_NAME> <LAST_NAME> for <PRODUCT> (<RELATIONSHIP>, <STATUS>; start <START_DATE>). Invite setup of digital channels.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an escalation note: Transaction Monitoring is Deprecated but was last executed 2025-09-15, Fully Automated, owned by Finance Team, Weekly cadence. Flag risk of dormant alerts.", "expected_scrub": "Write an escalation note: <PROCESS_NAME> is <PROCESS_STATUS> but was last executed <LAST_EXECUTED_DATE>, <AUTOMATION_STATUS>, owned by <RESP_TEAM>, <FREQUENCY> cadence.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Log a risk that Credit Scoring is Under Review (owner Customer Support, Fully Automated, On Demand, last run 2025-09-03). Note potential impact on downstream Loan Origination (Weekly, Semi-Automated, last run 2025-08-18, status Active).", "expected_scrub": "Log a risk that <PROCESS_NAME> is <PROCESS_STATUS> (owner <RESP_TEAM>, <AUTOMATION_STATUS>, <FREQUENCY>, last run <LAST_EXECUTED_DATE>). Note potential impact on <PROCESS_NAME> (<FREQUENCY>, <AUTOMATION_STATUS>, last run <LAST_EXECUTED_DATE>, status <PROCESS_STATUS>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a brief exec email summarizing status: Customer Onboarding (Deprecated, last 2025-09-05), KYC Verification (Deprecated, Monthly, last 2025-09-14), AML Screening (Deprecated, On Demand, last 2025-08-31), Fraud Detection (Active, Daily, last 2025-08-23).", "expected_scrub": "Draft a brief exec email summarizing status for <PROCESS_NAME> (<PROCESS_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an SOP step that, when Customer Complaint Handling (owner Customer Support, Fully Automated, On Demand) is Active (last run 2025-09-11), triage auto-tags tickets 'PRIORITY-CUST'.", "expected_scrub": "Write an SOP step that, when <PROCESS_NAME> (owner <RESP_TEAM>, <AUTOMATION_STATUS>, <FREQUENCY>) is <PROCESS_STATUS> (last run <LAST_EXECUTED_DATE>), triage auto-tags 'PRIORITY-CUST'.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a change request to align Data Backup (Under Review, Manual, On Demand, last run 2025-09-06) with Interest Calculation (Active, Fully Automated, Daily, last run 2025-09-02): ensure restore tests before EOM closes.", "expected_scrub": "Write a change request to align <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) with <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a controls snapshot comparing Regulatory Reporting (Under Review, Semi-Automated, On Demand, last 2025-09-16) and Internal Audit (Under Review, Fully Automated, Daily, last 2025-09-15).", "expected_scrub": "Compose a controls snapshot comparing <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) and <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Build a mini decision tree for Password Reset (Active, Semi-Automated, Daily, last executed 2025-08-22): if failures >3/day, auto-assign HR L2 and open P1.", "expected_scrub": "Build a mini decision tree for <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>): if failures >3/day, auto-assign L2 and open P1.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a weekly release plan for Mobile App Update (Active, Fully Automated, Weekly, last 2025-09-15) with a dependency check on Regulatory Reporting (Under Review, On Demand).", "expected_scrub": "Draft a weekly release plan for <PROCESS_NAME> (<PROCESS_STATUS>, <AUTOMATION_STATUS>, <FREQUENCY>, last <LAST_EXECUTED_DATE>) with dependency check on <PROCESS_NAME> (<PROCESS_STATUS>, <FREQUENCY>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Send an activation email for CUST-0023 (Line of Credit Agreement) to Emily Williams (emily.williams@hotmail.com) — Status Active; Amount 353,236.74.", "expected_scrub": "Send an activation email for <AGREEMENT_ID> (<AGREEMENT_TYPE>) to <PARTY_NAME> (<PARTY_EMAIL>) — Status <STATUS>; Amount <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a termination summary for CUST-0002 (Letter of Credit) — Michael Clark (michael.clark@gmail.com), Status Terminated, Amount 499,667.34.", "expected_scrub": "Create a termination summary for <AGREEMENT_ID> (<AGREEMENT_TYPE>) — <PARTY_NAME> (<PARTY_EMAIL>), Status <STATUS>, Amount <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "List active supplier agreements of outsourcing/related types: SUP-0012 (Outsourcing) 64,145.18; SUP-0036 (Outsourcing) 396,361.85; SUP-0039 (Outsourcing) 61,399.13; SUP-0034 (Services Supply) 237,777.45; SUP-0024 (Goods Supply) 106,623.59.", "expected_scrub": "List active supplier agreements of specified types with IDs and amounts: <AGREEMENT_ID> (<TYPE>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a pending-action tracker: CUST-0044 (Letter of Credit) 310,738.37; CUST-0045 (Bank Guarantee) 212,569.00; CUST-0046 (Personal Loan) 451,133.43; CUST-0054 (Safe Deposit Box) 203,538.70; CUST-0055 (Credit Card) 279,479.12; CUST-0056 (Overdraft) 492,879.62.", "expected_scrub": "Prepare a pending-action tracker: <AGREEMENT_ID> (<AGREEMENT_TYPE>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a risk note flagging high amounts: CUST-0028 (Active) 497,139.00; SUP-0016 (Expired) 496,703.00; SUP-0038 (Expired) 497,735.94; CUST-0002 (Terminated) 499,667.34.", "expected_scrub": "Draft a risk note flagging high amounts: <AGREEMENT_ID> (<STATUS>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a one-paragraph engagement snapshot for Emily Williams: Mortgage (Borrower, Active; last interaction 2025-08-11) and Current Account (Holder, Active; last 2025-07-30).", "expected_scrub": "Write a one-paragraph engagement snapshot for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>; last <LAST_INTERACTION_DATE>) and <PRODUCT_2> (...).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a micro-segmentation note for Mobile Banking Active users: include Robert Williams (last interaction 2025-08-08), Alex Garcia (2025-06-24), Katie Smith (2025-05-18). Suggest push notification campaign.", "expected_scrub": "Produce a micro-segmentation note for <PRODUCT> <STATUS> users: include <FIRST_NAME> <LAST_NAME> (last <LAST_INTERACTION_DATE>), ... Suggest a campaign.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a cross-sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Propose a mortgage health check offer.", "expected_scrub": "Draft a cross-sell note for <FIRST_NAME> <LAST_NAME>: <PRODUCT_A> (<RELATIONSHIP>, <STATUS>), <PRODUCT_B> (<RELATIONSHIP>, <STATUS>), <PRODUCT_C> (<RELATIONSHIP>, <STATUS>). Propose an offer.", "entities": null, "notes": null}
{"id": "prompts_10_12_16:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose gentle nudges to re-engage Online Banking users now Inactive: John Taylor (last 2025-03-19), Katie Taylor (2025-05-04).", "expected_scrub": "Compose gentle nudges to re-engage <PRODUCT> users now <STATUS>: <FIRST_NAME> <LAST_NAME> (last <LAST_INTERACTION_DATE>), ...", "entities": null, "notes": null}
{"id": "prompts_10_12_16:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a relationship matrix entry for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).", "expected_scrub": "Create a relationship matrix entry for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>), <PRODUCT_2> (...), <PRODUCT_3> (...).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a compliance linkage note: Emily Jones has Electronic Banking Agreement CUST-0050 Active (Amount 35,037.61). Product links show Mobile Banking (User, Active) and Current Account (Holder, Active).", "expected_scrub": "Compose a compliance linkage note: <FIRST_NAME> <LAST_NAME> has <AGREEMENT_ID> <STATUS> (Amount <AMOUNT>). Product links show <PRODUCT_1> (<RELATIONSHIP>, <STATUS>) and <PRODUCT_2> (<RELATIONSHIP>, <STATUS>).", "entities": null, "notes": null}
{"id": "prompts_10_12_16:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_10_12_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a friendly welcome email to Jane Smith for her Mortgage (Borrower, Active; start 2024-09-30). Invite her to set up Online/Mobile Banking.", "expected_scrub": "Write a friendly welcome email to <FIRST_NAME> <LAST_NAME> for <PRODUCT> (<RELATIONSHIP>, <STATUS>; start <START_DATE>). Invite setup of digital channels.", "entities": null, "notes": null}
