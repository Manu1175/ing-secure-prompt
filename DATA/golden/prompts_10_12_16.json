[
  {
    "text": "Write an escalation note: Transaction Monitoring is Deprecated but was last executed 2025-09-15, Fully Automated, owned by Finance Team, Weekly cadence. Flag risk of dormant alerts.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Log a risk that Credit Scoring is Under Review (owner Customer Support, Fully Automated, On Demand, last run 2025-09-03). Note potential impact on downstream Loan Origination (Weekly, Semi-Automated, last run 2025-08-18, status Active).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a brief exec email summarizing status: Customer Onboarding (Deprecated, last 2025-09-05), KYC Verification (Deprecated, Monthly, last 2025-09-14), AML Screening (Deprecated, On Demand, last 2025-08-31), Fraud Detection (Active, Daily, last 2025-08-23).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Write an SOP step that, when Customer Complaint Handling (owner Customer Support, Fully Automated, On Demand) is Active (last run 2025-09-11), triage auto-tags tickets 'PRIORITY-CUST'.",
    "labels": [
      "BIC",
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Write a change request to align Data Backup (Under Review, Manual, On Demand, last run 2025-09-06) with Interest Calculation (Active, Fully Automated, Daily, last run 2025-09-02): ensure restore tests before EOM closes.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Compose a controls snapshot comparing Regulatory Reporting (Under Review, Semi-Automated, On Demand, last 2025-09-16) and Internal Audit (Under Review, Fully Automated, Daily, last 2025-09-15).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Build a mini decision tree for Password Reset (Active, Semi-Automated, Daily, last executed 2025-08-22): if failures >3/day, auto-assign HR L2 and open P1.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a weekly release plan for Mobile App Update (Active, Fully Automated, Weekly, last 2025-09-15) with a dependency check on Regulatory Reporting (Under Review, On Demand).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Send an activation email for CUST-0023 (Line of Credit Agreement) to Emily Williams (emily.williams@hotmail.com) — Status Active; Amount 353,236.74.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Create a termination summary for CUST-0002 (Letter of Credit) — Michael Clark (michael.clark@gmail.com), Status Terminated, Amount 499,667.34.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "List active supplier agreements of outsourcing/related types: SUP-0012 (Outsourcing) 64,145.18; SUP-0036 (Outsourcing) 396,361.85; SUP-0039 (Outsourcing) 61,399.13; SUP-0034 (Services Supply) 237,777.45; SUP-0024 (Goods Supply) 106,623.59.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Prepare a pending-action tracker: CUST-0044 (Letter of Credit) 310,738.37; CUST-0045 (Bank Guarantee) 212,569.00; CUST-0046 (Personal Loan) 451,133.43; CUST-0054 (Safe Deposit Box) 203,538.70; CUST-0055 (Credit Card) 279,479.12; CUST-0056 (Overdraft) 492,879.62.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a risk note flagging high amounts: CUST-0028 (Active) 497,139.00; SUP-0016 (Expired) 496,703.00; SUP-0038 (Expired) 497,735.94; CUST-0002 (Terminated) 499,667.34.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write a one-paragraph engagement snapshot for Emily Williams: Mortgage (Borrower, Active; last interaction 2025-08-11) and Current Account (Holder, Active; last 2025-07-30).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Produce a micro-segmentation note for Mobile Banking Active users: include Robert Williams (last interaction 2025-08-08), Alex Garcia (2025-06-24), Katie Smith (2025-05-18). Suggest push notification campaign.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a cross-sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Propose a mortgage health check offer.",
    "labels": []
  },
  {
    "text": "Compose gentle nudges to re-engage Online Banking users now Inactive: John Taylor (last 2025-03-19), Katie Taylor (2025-05-04).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Create a relationship matrix entry for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).",
    "labels": []
  },
  {
    "text": "Compose a compliance linkage note: Emily Jones has Electronic Banking Agreement CUST-0050 Active (Amount 35,037.61). Product links show Mobile Banking (User, Active) and Current Account (Holder, Active).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write a friendly welcome email to Jane Smith for her Mortgage (Borrower, Active; start 2024-09-30). Invite her to set up Online/Mobile Banking.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  }
]